packages/driver/test/cypress/integration/commands/querying_spec.js

```diff
@@ -1567,13 +1567,13 @@ describe('src/cy/commands/querying', () => {
     it('will not find script elements', () => {
       cy.$$('<script>// some-script-content </script>').appendTo(cy.$$('body'))
 
-      cy.contains('some-script-content').should('not.match', 'script')
+      cy.contains('some-script-content').should('not.exist')
```

Comment:
As we use `innerText` now, contains cannot find the tag at all because text in script tag is invisible